docs/en/Deploy-skywalking-agent.md

```diff
@@ -39,5 +39,10 @@ CATALINA_OPTS="$CATALINA_OPTS -javaagent:/path/to/skywalking-agent/skywalking-ag
 - Tomcat 8
 Change the first line of `tomcat/bin/catalina.sh`.
 ```shell
-set "CATALINA_OPTS=-javaagent:E:\apache-tomcat-8.5.20\skywalking-agent\skywalking-agent.jar"
-```
\ No newline at end of file
+set "CATALINA_OPTS=-javaagent:/path/to/skywalking-agent/skywalking-agent.jar"
+```
+- JAR file
+Change the boot command
```

Comment:
I want to recommend you to change your English expression.

- My suggestion
Add `-javaagent` argument to command line in which you start your app. And make sure to add it before the -jar argument.

Reply:
ok, thanks